### DISCUSSION TITLE
My NextJs useState Totally stopped working

### DISCUSSION THREAD

**Discussion by mVedr** [vercel/next.js#56790] [Help]
### Summary
```
'use client'

import React, { useState } from 'react';
import Image from 'next/image';
export default function Home() {
  const [count,setCount] = useState<number>(0)
  const add = ()=>{
    setCount(prevCount => prevCount + 1);
  }
  return (
    <main className="flex min-h-screen flex-col items-center justify-between p-24">
      <h1>{count}</h1>
      <button onClick={add}>+</button>
    </main>
  )
}
```
This is a simple counter app, but the count is not being updated ...This is what my package.json looks like :

(https://i.stack.imgur.com/eROKr.png)

This is a simple counter app, but the count is not being updated ...

### Additional information

```
There are no errors whatsoever, I am so confused, i am an old user , but trying to migrate to 13 version.
```

**Comment by devjiwonchoi** [comment]
Try to upgrade your dependencies. For example, `pnpm up`.

**Comment by icyJoseph** [comment]
@mVedr there was an issue with Node versions earlier than 16.14. You must have at least 16.14, please try upgrading that.

Also some versions under early Node 17, seem to have had the same issue.

**Reply by mVedr** [reply]
Hi yes , it is working now after updating the version, thanks.

**Reply by icyJoseph** [reply]
Awesome, please don't forget to mark as answered. Back when this issue was first noticed, it was very hard to debug :( 

I was however included in the migration guide:

- https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#nodejs-version
